packages/help-center/src/components/back-to-top-button.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button } from '@automattic/components';
+import { useCallback, useEffect, useState } from '@wordpress/element';
+import { Icon, arrowUp } from '@wordpress/icons';
+import { useI18n } from '@wordpress/react-i18n';
+import classnames from 'classnames';
+import type { FC, RefObject } from 'react';
+import './back-to-top-button.scss';
+
+type Props = { container: RefObject< HTMLDivElement > };
+
+export const BackToTopButton: FC< Props > = ( { container } ) => {
+	const { __ } = useI18n();
+	const [ visible, setVisible ] = useState( false );
+
+	const SCROLL_THRESHOLD = 400;
+
+	const scrollCallback = useCallback( () => {
+		const containerNode = container.current;
+
+		if ( containerNode ) {
+			if ( containerNode.scrollTop > SCROLL_THRESHOLD ) {
+				setVisible( true );
+			} else {
+				setVisible( false );
+			}
+		}
+	}, [ container ] );
+
+	useEffect( () => {
+		const containerNode = container?.current;
+
+		if ( containerNode ) {
+			containerNode.addEventListener( 'scroll', scrollCallback );
+
+			return () => {
+				if ( containerNode ) {
```

Comment:
Do we need to check again for `containerNode`? I feel like we're already covered here, but maybe you found that this is needed?